Wiki-Template-Challenge-Solution.md

```diff
@@ -1,84 +1,98 @@
 :triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :memo:
 
 # :checkered_flag: Problem Explanation:
+
 Explain what is asked, in an easy to understand and detailed way.
 
 #### Relevant Links
+
 - [Link Title 1](http://example.com)
 - [Link Title 2](http://example.com)
 - [Link Title 3](http://example.com)
 
 ## :speech_balloon: Hint: 1
+
 Add hint 1 here.
 
-> *try to solve the problem now*
+> _try to solve the problem now_
 
 ## :speech_balloon: Hint: 2
+
 Add hint 2 here.
 
-> *try to solve the problem now*
+> _try to solve the problem now_
 
 ## :speech_balloon: Hint: 3
+
 Add hint 3 here.
 
-> *try to solve the problem now*
+> _try to solve the problem now_
 
 ## Spoiler Alert!
+
 [![687474703a2f2f7777772e796f75726472756d2e636f6d2f796f75726472756d2f696d616765732f323030372f31302f31302f7265645f7761726e696e675f7369676e5f322e676966.gif](https://files.gitter.im/FreeCodeCamp/Wiki/nlOm/thumb/687474703a2f2f7777772e796f75726472756d2e636f6d2f796f75726472756d2f696d616765732f323030372f31302f31302f7265645f7761726e696e675f7369676e5f322e676966.gif)](https://files.gitter.im/FreeCodeCamp/Wiki/nlOm/687474703a2f2f7777772e796f75726472756d2e636f6d2f796f75726472756d2f696d616765732f323030372f31302f31302f7265645f7761726e696e675f7369676e5f322e676966.gif)
 
 **Solution ahead!**
 
 ## :beginner: Basic Code Solution:
 
-```js
+```javascript
 // write solution code here
 ```
 
 ### Code Explanation:
+
 Explain solution here and add any relevant links
 
 #### Relevant Links
+
 - [Link Title 1](http://example.com)
 - [Link Title 2](http://example.com)
 - [Link Title 3](http://example.com)
 
 ## :sunflower: Intermediate Code Solution:
 
-```js
+```javascript
 // write solution code here
 ```
 
 ### Code Explanation:
+
 Explain solution here and add any relevant links
 
 #### Relevant Links
+
 - [Link Title 1](http://example.com)
 - [Link Title 2](http://example.com)
 - [Link Title 3](http://example.com)
 
 ## :rotating_light: Advanced Code Solution:
 
-```js
+```javascript
```

Comment:
Nothing, Atom changed it automatically, I reverted some changes but left that as it. It is not in bad form after all.